packages/worker/src/tasks/syncPackageIndexBatch.ts

```diff
@@ -31,9 +31,13 @@ async function syncPackage(name: string) {
         name: name,
         latestVersion: statsAndMetadata.latestVersion,
         description: statsAndMetadata.description,
+        fullDescription: statsAndMetadata.fullDescription,
+        maintainers: statsAndMetadata.maintainers,
+        tags: statsAndMetadata.tags,
```

Comment:
nit: I was thinking about keeping the `keywords` naming instead of `tags`. What's your take?